tests/callbacks/test_TimerListener.cpp

```diff
@@ -207,4 +208,30 @@ TEST(RegionTimerListener, traits_when_used_with_enqueued_event_timer)
     >);
 }
 
+//! @test Check the parallel for timer listener when used with @ref EnqueuedEventWithLaunchTimer.
+TEST_F(TimerListenerTest, with_launch)
+{
```

Comment:
Move the test a bit up, before the region timer tests ?

Reply:
OK. Better placed indeed!